src/resources/help/en_US/relnotes3.2.3.html

```diff
@@ -31,6 +30,7 @@
   <li>Updated the Ohio Supercomputing Center host profiles.</li>
   <li>Chombo plugin now supports anisotropic refinements.</li>
   <li>The WData database reader was added.</li>
+  <li>Moved divergence, curl, gradient, and laplacian functions from mics. to vector submenu. </li>
```

Comment:
How about:
Moved the *divergence*, *curl*, *gradient*, and *Laplacian* functions from the *Miscellaneous* submenu to the *Vector* submenu within the *Insert function...* menu in the Expressions window.

Reply:
Sorry about that! I'll try to write more descriptive sentences in the release notes in the future! I've made the change you asked for.